uint32 date #GPS date
uint32 time #GPS time
int32 longitude #unit in degree*10^7
int32 latitude  #unit in degree*10^7
int32 height_above_sea #unit in mm 
float32 velocity_north #unit in cm/s
float32 velocity_east #unit in cm/s
float32 velocity_ground #unit in cm/s
